.build/parent-pom-template.xml

```diff
@@ -671,7 +671,7 @@
       <dependency>
         <groupId>org.apache.cassandra</groupId>
         <artifactId>cassandra-all</artifactId>
-        <version>4.1-alpha2-SNAPSHOT</version>
```

Comment:
interesting

Reply:
yeah, this surprised me too!  Ultimately it wasn't being used because I don't think it was used until now.